core/src/main/contraband/librarymanagement.json

```diff
@@ -726,6 +726,19 @@
         { "name": "devConnection", "type": "String?",     "default": "None", "since": "0.0.1" }
       ]
     },
+    {
+      "name": "LicenseInfo",
+      "namespace": "sbt.librarymanagement",
+      "target": "Scala",
+      "type": "record",
+      "doc": "Basic license information for a project module",
+      "fields": [
+        { "name": "name",          "type": "String",      "default": "\"\"",                        "since": "2.0.0" },
+        { "name": "url",           "type": "java.net.URL","default": "new java.net.URL(\"http:\")", "since": "2.0.0" },
```

Comment:
So contraband has changed its behaviour since I opened this PR and its now asking for default values on fields. While for `String` its trivial, with `java.net.URL` doing `new URL("http:")` seems to be the minimal way I can construct the type

Reply:
did you retry not setting `since`? Maybe it does not require setting the `default` field if `since` is not set...